packages/vehicle-rental-overlay/src/StationPopup.tsx

```diff
@@ -1,146 +0,0 @@
-import { Styled as BaseMapStyled } from "@opentripplanner/base-map";
```

Comment:
This file was involved in commit https://github.com/opentripplanner/otp-ui/pull/472/commits/671c6aeeafe3c0bddebf80cb1f43c96d4e45f2a8 which is a `feat` commit, are we expecting to issue a feature release for the vehicle rental overlay package?

Reply:
I think we should -- this is quite a change as it brings in new dependencies and no longer exports the popup contents itself.